tests/unit/tools/executor/test_tool_executor.py

```
@@ -387,7 +388,7 @@ def test_executor_unknown_tool(
     """
     _stub_logger(monkeypatch, fake_logger)
 
-    def raise_value_error(tools: str | None, action: str) -> Never:
+    def raise_value_error(tools: str | None, action: str, **_kwargs: object) -> Never:
```

Ruff (0.15.6)

[warning] Unused function argument: `tools`

(ARG001)